test/unit/testExecutorsDealing.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const initSteps = require('app/core/initSteps');
+const {expect, assert} = require('chai');
+
+describe('Executors-Applying', function () {
+    let ctx;
+    const ExecsDealing = initSteps([__dirname + '/../../app/steps/action/', __dirname + '/../../app/steps/ui']).ExecutorsDealingWithEstate;
+    describe('pruneFormData', () => {
+
+        it('test that isApplying flag is deleted when executor is not applying', () => {
+            ctx = {
```

Comment:
It would be more accurate to call this variable `data` as it's the data for an executor not the context